tannerb9/MasksNowTN-New
# tb-Contact

# Description
Created a v-footer for Contact component. Included the company name and the three icons for FB, IG, and Email that each link to their respective site/app. I centered each but might be better to have each on the same line on opposite ends. Currently colored black and white until site style/colors finalized.

## Type of change

# Testing Instructions
Run the app and test all icons to ensure they go to the correct pages or open your email app(for email only)
